client/src/modules/list/components/ItemsList/ItemsList.jsx

```diff
@@ -79,8 +74,8 @@ class ItemsList extends Component {
                 name={item.name}
                 toggleItem={this.toggleItem}
                 voteForItem={this.voteForItem(item)}
-                votesCount={item.voterIds.length}
-                whetherUserVoted={item.voterIds.includes(userId)}
+                votesCount={item.votesCount}
+                whetherUserVoted={item.didCurrentUserVoted}
```

Comment:
maybe it will be easier to remember if it will be the same shorter name ? e.g. didUserVote={item.didUserVote}. If you will decide to change it - change it in all places.

Reply:
or just `isVoted`